corehq/apps/reports/standard/cases/case_list_explorer.py

```diff
@@ -176,8 +182,11 @@ def headers(self):
     @property
     def rows(self):
         self.track_search()
-        data = (wrap_case_search_hit(row) for row in self.es_results['hits'].get('hits', []))
-        return self._get_rows(data)
+
+        with self.profiler.timing_context("Retrieving rows"):
+            data = (wrap_case_search_hit(row) for row in self.es_results['hits'].get('hits', []))
+            with self.profiler.timing_context("Parsing rows"):
```

Comment:
nit: preparing data for display?

Reply:
I like that. I'll update.

Reply:
https://github.com/dimagi/commcare-hq/pull/35830/commits/8451b535f2f516a16950738006212279e5c5ff42